mmdet/core/loss/losses.py

```diff
@@ -127,3 +127,36 @@ def _expand_binary_labels(labels, label_weights, label_channels):
     bin_label_weights = label_weights.view(-1, 1).expand(
         label_weights.size(0), label_channels)
     return bin_labels, bin_label_weights
+
+
+def iou_loss(pred, target, reduction='mean'):
```

Comment:
Please refer to https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/bbox/geometry.py#L4 for more efficient implementation.